src/isilon_hadoop_tools/directories.py

```diff
@@ -61,8 +61,14 @@ def create_directories(self, directories, setup=None, mkdir=None, chmod=None, ch
             setup(zone_root, hdfs_root, zone_hdfs)
         for directory in directories:
             path = posixpath.join(zone_hdfs, directory.path.lstrip(posixpath.sep))
-            LOGGER.info("mkdir '%s%s'", zone_root, path)
-            (mkdir or self.onefs.mkdir)(path, directory.mode, zone=self.onefs_zone)
+            try:
+                LOGGER.info("mkdir '%s%s'", zone_root, path)
+                (mkdir or self.onefs.mkdir)(path, directory.mode, zone=self.onefs_zone)
+            except IsilonHadoopToolError as exc:
```

Comment:
`directory_path_already_exists_error` is only guaranteed to exist on `onefs.APIError`:
``` python
import isilon_hadoop_tools.onefs
```
```suggestion
            except isilon_hadoop_tools.onefs.APIError as exc:
```